aspnetcore/fundamentals/metapackage.md

```diff
@@ -12,7 +12,10 @@ ms.topic: article
 uid: fundamentals/metapackage
```

Comment:
metapackage-all?

Reply:
No need to change the file name and the UID.